WalletWasabi.Fluent/Behaviors/PrivacyWarningFadeOutBehavior.cs

```diff
@@ -0,0 +1,54 @@
+using Avalonia;
+using Avalonia.Animation;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using ReactiveUI;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.Models.Transactions;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class PrivacyWarningFadeOutBehavior : AttachedToVisualTreeBehavior<Control>
+{
+	private const string FadeOutClassName = "fadeout";
+
+	public static readonly StyledProperty<IEnumerable<PrivacyWarning>> PreviewWarningsProperty =
+		AvaloniaProperty.Register<PrivacyWarningFadeOutBehavior, IEnumerable<PrivacyWarning>>(nameof(PreviewWarnings));
+
+	public IEnumerable<PrivacyWarning> PreviewWarnings
+	{
+		get => GetValue(PreviewWarningsProperty);
+		set => SetValue(PreviewWarningsProperty, value);
+	}
+
+	protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
+	{
+		if (AssociatedObject?.DataContext is not PrivacyWarning current)
+		{
+			return;
+		}
+
+		this.WhenAnyValue(x => x.PreviewWarnings)
+			.WhereNotNull()
+			.Throttle(TimeSpan.FromMilliseconds(250), RxApp.MainThreadScheduler)
```

Comment:
I can still repro the flickering bug.
![fadebug2](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/4cb60208-d49a-44a6-85a6-fece263bdfbf)